handbook/openid-connect/openid-connect.md

```diff
@@ -43,14 +43,16 @@ You can configure SystemLink to use OpenID Connect to authorize users. This enab
 10. To enable OpenID Connect as the only login option, go back to **NI Web Server Configuration** > **Authentication** and disable **Log in as users controlled by the web server**.
 
 <figure>
-  <img src="../oidc-webserver.png" width="500" />
+  <img src="../../img/oidc-webserver.png" width="500" />
   <figcaption>Enable OpenID Connect in NI Web Server</figcaption>
 </figure>
 
 ## OpenID Configuration Files in SystemLink Server
 
 There are three files that you must create to connect your SystemLink server to an OpenID Connect provider: `[provider-dns].conf`, `[provider-dns].client`, and `[provider-dns].provider`. The `[provider-dns]` portion of each filename must be the URL-encoded fully qualified domain name.
 
+Refer to [example-slclient-config](https://github.com/ni/systemlink-operations-handbook/tree/master/handbook/openid-connect/example-slclient-config) for examples of each of these files.
```

Comment:
Do you think it would be beneficial to link directly to the example files (within the handbook) instead of users needing to switch to GitHub? As in, you could do something like:

```md
Refer to [slsso-runtime.com.client](/openid-connect/example-slclient-config/slsso-runtime.com.client), [slsso-runtime.com.conf](/openid-connect/example-slclient-config/slsso-runtime.com.conf), and [slsso-runtime.com.provider](/openid-connect/example-slclient-config/slsso-runtime.com.provider) for examples of each of these files.
```

And each of the links will download the associated file.

Reply:
Otherwise, I think it defeats the purpose of having the files within the handbook directory and should instead be in an examples directory or something outside of the handbook directory.

Reply:
This has been moved outside of the handbook because its best just to send the user directly to GitHub at this point.